### DISCUSSION TITLE
Koishi 闲聊专用楼 (/≧▽≦)/

### DISCUSSION THREAD

**Discussion by ilharp** [koishijs/koishi#246] [General]
<div align="center">
  <img width="160" src="https://koishi.js.org/koishi.png" alt="logo">
  <h1 id="koishi">Koishi 闲聊专用楼</h1>
</div>

## 这是……

这里是 Koishi 的闲聊专用楼 (/≧▽≦)/

在这里，你可以分享 Koishi 的使用体验、和大家交流 Koishi 的使用心得，或是和大家分享自己在开发过程中的一些技巧与经验。

请畅所欲言 (/≧▽≦)/

## 这不是……

- 使用教程：请查阅 [官网的文档](https://koishi.js.org/)。

（当然，对官网的文档有什么建议的话可以在这里说~

- 问题追踪：请前往 [Issues](https://github.com/koishijs/koishi/issues) 反馈问题。

- 功能建议/疑难解答：请前往 [Discussions](https://github.com/koishijs/koishi/discussions) 并准确选择板块发帖。

## 请注意

- **在本楼的发言并不会得到任何人快速的回复。** 如果你遇到了较为紧急的状况，请前往 [Issues](https://github.com/koishijs/koishi/issues) 反馈。

- 为了 Discussions 版整体环境考虑，请不要在本楼之外的地方发表 **占用问题号（Issue/PR/Discussion）** 的水贴。

---

那么，请愉快地聊天吧！聊天时请注意 **热情、友善** 哦~

**Comment by dragon-fish** [comment]
从此进入版聊时代

**Comment by kecrily** [comment]
没了群现在我该怎么才能和大佬们当上朋友呢

**Comment by dragon-fish** [comment]
建议：使用discord等可以利用用户组分隔开可信用户、维护组用户、闲聊用户的IM软件

**Comment by Anillc** [comment]
要不要搭个koishi论坛（

**Reply by ilharp** [reply]
个人认为目前的情况来看 Discussions 版已经足够使用

**Reply by dragon-fish** [reply]
论坛在国内的备案可能很成问题，我也觉得 Discussions 够用了

**Reply by Anillc** [reply]
备案干啥，丢国外服务器不就行啦

**Reply by dragon-fish** [reply]
论坛更容易导致人多口杂隔墙有耳吧（

**Reply by Anillc** [reply]
论坛怕什么隔墙有耳，本来就是public的（

**Comment by dragon-fish** [comment]
~~讨论版居然是实时刷新的，有丶酷~~

**Reply by NWYLZW** [reply]
牛哇

**Reply by Anillc** [reply]
其实issue这些都是实时刷新的（

**Reply by dragon-fish** [reply]
可能是我以前网不好

**Reply by ilharp** [reply]
Gh 一直都有实时刷新的功能吧

**Reply by NWYLZW** [reply]
那么肯定webscoket吧

**Reply by dragon-fish** [reply]
这不比贴吧刺激

**Reply by ilharp** [reply]
网不好的区域确实可能刷不到

**Reply by NWYLZW** [reply]
好像也不是实时的，必须刷新才行

**Reply by dragon-fish** [reply]
> 
> 好像也不是实时的，必须刷新才行

没有啊，我这里都是直接刷新的

**Reply by ilharp** [reply]
我这边是实时的

**Reply by NWYLZW** [reply]
> > 好像也不是实时的，必须刷新才行
> 
> 没有啊，我这里都是直接刷新的

偶尔会有点问题，现在倒是好了

**Reply by dragon-fish** [reply]
除了邮件轰炸以外什么都好（）

**Reply by ilharp** [reply]
没有看到 `ws` 连接，推测可能是轮询，但好像也不是定长

**Reply by ilharp** [reply]
很奇妙

**Reply by NWYLZW** [reply]
我邮件倒是关了

**Reply by kecrily** [reply]
> 除了邮件轰炸以外什么都好（）

已经被轰炸了

**Reply by NWYLZW** [reply]
在*回复*的时候，是不会刷新的
我这边

**Reply by dragon-fish** [reply]
> 
> 
> 在_回复_的时候，是不会刷新的
> 我这边

打开编辑框的时候，该条讨论串不会刷新

**Reply by NWYLZW** [reply]
是的，我也发现了

> > 在_回复_的时候，是不会刷新的
> > 我这边
> 
> 打开编辑框的时候，该条讨论串不会刷新

**Reply by Anillc** [reply]
邮件的话拉到页面底部，unsubscribe就行了（，不过如果重新回复，他又会自动给你subscribe上

**Reply by NWYLZW** [reply]
这个楼盖的真高啊

**Reply by ilharp** [reply]
好事

**Reply by dragon-fish** [reply]
~~我要关闭邮件了，好 TM 吵~~

**Reply by kecrily** [reply]
邮件 duangduangduang 地响

**Reply by ilharp** [reply]
😄

**Reply by cyans-nya** [reply]
邮箱响了吧（

**Comment by NWYLZW** [comment]
我觉得我的TODO插件可以加个功能
检测某个项目里面有哪些TODO
然后按照指定的格式分配给用户

例如
```ts
// TODO [x] <onebot:2284672637|discord:uid> 对接后台接口
//   * 登陆接口
//   * 注册接口
```
似乎很棒诶

**Comment by NWYLZW** [comment]
假如我的插件依赖ppt插件
那我应该把它放在peerDep中吗
感觉不太对的样子
![image](https://user-images.githubusercontent.com/51358815/115986851-e0f1c680-a5e4-11eb-86d4-ed705c46ef94.png)

**Comment by plasmatank** [comment]
来了 :D

**Comment by ilharp** [comment]
![FE2112BBC8DD7EA727B5125FA3BE167F](https://user-images.githubusercontent.com/20179549/116077013-9006ce80-a6c7-11eb-96d3-6778de5a89d2.jpg)

**Comment by ilharp** [comment]
![image](https://user-images.githubusercontent.com/20179549/116865856-044bef80-ac3d-11eb-8507-9de62491cbdc.png)

**Comment by borraken** [comment]
好耶

**Comment by ilharp** [comment]
![image](https://user-images.githubusercontent.com/20179549/117533169-f1129880-b01d-11eb-848b-ed6f04a155d6.png)

**Comment by idlist** [comment]
Q：为什么精品问答集不更新了？

A：因为我好久没写 bot 了

**Comment by idlist** [comment]
关于近期新环境包 [cosmotype](https://github.com/cosmotype/cosmotype):

![cosmotipo](https://user-images.githubusercontent.com/29368226/164252144-760becba-4fc1-4157-801b-2e892fb23497.png)

cosmo~~type~~tipo

**Comment by liyuier** [comment]
https://github.com/cordiverse/cordis
cordis 时代挖坟，原来作者去给 ds 写 agent 了

**Reply by NWYLZW** [reply]
我日，这也能挖到坟

**Reply by liyuier** [reply]
毕竟 Koishi 还是挺有名的嘛
